debian/copyright

```diff
@@ -0,0 +1,26 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: cometvisu
+Source: https://github.com/CometVisu/CometVisu
+
+Files: *
+Copyright: Christian Mayer <CometVisu@ChristianMayer.de>
+           Tobias Bräutigam <tbraeutigam@gmail.com>
+           Michael Markstaller <devel@wiregate.de> / Elaborated Networks GmbH
```

Comment:
On 03.07.2016 21:08, Christian Mayer wrote:
> +Copyright: Christian Mayer <CometVisu@ChristianMayer.de>
>> +           Tobias Bräutigam <tbraeutigam@gmail.com>
>> +           Michael Markstaller <devel@wiregate.de> / Elaborated Networks GmbH
> 
> Makki is not associated with Elaborated Networks anymore.


Thats correct though I don't really care in terms of formulating the (C)
- I wont change it just for this..
If you do changes, feel free to use

Michael Markstaller <michael@markstaller.de>


-- 
Für Rückfragen stehe ich jederzeit gerne zur Verfügung.

Mit freundlichen Grüssen

Michael Markstaller
Softwareentwicklung & IT-Dienstleistungen

Am Birkengarten 19b
82024 Taufkirchen
Tel: +49-89-21553825, Mobile: +49-179-2080789
USt-ID: DE293191950